cpp/src/groupby/sort/group_rank_scan.cu

```diff
@@ -321,6 +320,8 @@ std::unique_ptr<column> group_rank_to_percentage(rank_method const method,
                                 : one_normalized(r, count);
                      });
   }
+
+  ranks->set_null_count(0);
```

Comment:
It would not matter because this call will _invalidate_ the null-count: https://github.com/rapidsai/cudf/pull/13149/files#diff-0278ebe1facc446e3ce63980099f1620735c28a95aac43b17eb1e23d235a5e90R280
```
auto mutable_ranks = ranks->mutable_view();
```
So we need to set it back somewhere after that line.

Reply:
Ah right good call. Thanks.